packages/showcases/svelte/stories/components/navigation/VtmnSearch/VtmnSearch.stories.svelte

```diff
@@ -37,7 +37,8 @@
   <VtmnSearch
     {...args}
     on:input={(e) => console.log(e.target.value)}
-    on:search={(event) => alert('Search event fired ! :' + event.detail.text)}
+    on:search={(event) =>
+      console.log('Search event fired ! :' + event.detail.text)}
```

Comment:
Is it ok to have a console.log in the code ?

Reply:
Since it's a showcase for the UI, I'm ok with the fact this functional event could be more discreet. A console.log is fine, an addon action could be fine too indeed.